lib/configure.js

```diff
@@ -66,6 +68,13 @@ travis.users.sync.post = promisify(travis.users.sync.post);
 // when creating GitHub tokens.
 var noteUrl = 'https://github.com/mozilla/oghliner';
 
+// The message currently displayed with a spinner, if any.  Most async
+// operations don't need to set this, but occasionally one displays a message
+// and then calls a function that displays another message before
+// the operation is complete.  In that case, the function needs to stop
+// the spinner for the first message before it displays the second message.
+var spinnerMessage;
```

Comment:
> Could you create a function that does this automatically? Maybe by wrapping cli.spinner.

I've been looking into adopting the one you created for *integrate*, but this case is trickier, because we want to abort the current spinner message, display a new one, then re-display the original one after the new one is done.

Which is not ideal output anyway, since it's a bit strange to leave the original spinner message hanging, without either a checkmark/done or x-mark/error indicator. So I've been thinking about a different design that displays only a single spinner message but also displays a note if the token creation required deletion of an existing token (like .travis.yml file creation displays a warning if it had to create the file), something like:

```
✓ Deleting temporary GitHub token for getting Travis token… done!
✓ Creating permanent GitHub token for Travis to push to the repository… done!

ℹ You had an existing token for this app, so we deleted and recreated it.
```

My first attempt at this is in e642a27, but it doesn't look quite right yet, as it results in:

```
✓ Deleting temporary GitHub token for getting Travis token… done!

ℹ You had an existing token for this app, so we deleted and recreated it.

✓ Creating permanent GitHub token for Travis to push to the repository… done!
```